actix-server/src/accept.rs

```diff
@@ -83,9 +83,59 @@ struct Accept {
     handles: Vec<WorkerHandleAccept>,
     srv: Server,
     next: usize,
+    avail: Availability,
     backpressure: bool,
 }
 
+/// Array of u128 with every bit as marker for a worker handle's availability.
+struct Availability([u128; 4]);
```

Comment:
any reason to hand roll this as opposed to using one of the structures from `bitvec` ?

Reply:
We may want to replace this with Arc<[AtomicU64]> when #327 progress. The current one is mainly to fix the backpressure bug and find a suitable API for it.